src/pages/GaragePage.js

```diff
@@ -0,0 +1,16 @@
+import BasePage from "./BasePage.js";
+
+export default class GaragePage extends BasePage {
+    constructor(page) {
+        super(page, '/panel/garage',
+            page.locator('app-panel-layout' , {has : page.locator('button', {hasText: 'Add car'})}));
+    }
+
+    async getMainTitle() {
+        await this._page.title();
+    }
+
+    async getPageUrl() {
+        await this._page.url();
+    }
```

Comment:
all these methods should be in BasePage

Reply:
but I recommend to use page element for assertion, for example "Add car" button

Reply:
I'll fixed all comments in next pull.